js_modules/dagit/packages/core/src/ui/PageSection.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components/macro';
+
+import {ColorsWIP} from './Colors';
+
+export const PageSection = styled.div`
```

Comment:
Ahh I figured this would have some padding, etc.

If it's just a divider what do you think of creating a `PageDivider` that renders an `<hr/>` that we style to be a 1px keyline gray line?

Reply:
I couldn't really apply padding to these because the tables have to be full-width, and I didn't want to go messing around with negative margins.